src/styles/global/colors/_index.scss

```diff
@@ -0,0 +1,15 @@
+// Color palette -- All colors variables should start with $color-
+// ---------------------------------------------------
+
+$color-black: #000;
+$color-white: #fff;
+$color-red: #ff0000;
+
+// Add each variable to the following :export for docz
+// ---------------------------------------------------
+
+:export {
+  color_black: $color-black;
+  color_red: $color-red;
+  color_white: $color-white;
+}
```

Comment:
While, I'm good with merging this as is, consider what we discussed — a way to achieve a similar result without duplicating (and renaming) variables.